<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>873</width>
    <height>685</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>861</width>
      <height>271</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="dataTab">
     <attribute name="title">
      <string>Data</string>
     </attribute>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>851</width>
        <height>201</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QGroupBox" name="databaseBox">
         <property name="title">
          <string>Database connection configuration</string>
         </property>
         <widget class="QWidget" name="formLayoutWidget">
          <property name="geometry">
           <rect>
            <x>9</x>
            <y>19</y>
            <width>401</width>
            <height>162</height>
           </rect>
          </property>
          <layout class="QFormLayout" name="databaseFormLayout">
           <property name="fieldGrowthPolicy">
            <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
           </property>
           <property name="labelAlignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="dbhostLabel">
             <property name="text">
              <string>Host</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="dbhostLineEdit">
             <property name="inputMask">
              <string/>
             </property>
             <property name="text">
              <string>localhost</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="dbnameLabel">
             <property name="text">
              <string>Database</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="dbnameLineEdit">
             <property name="inputMask">
              <string/>
             </property>
             <property name="text">
              <string>avignon_projets</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="dbuserLabel">
             <property name="text">
              <string>User</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="dbuserLineEdit">
             <property name="inputMask">
              <string/>
             </property>
             <property name="text">
              <string>claire</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="dbpasswordLabel">
             <property name="text">
              <string>Password</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="dbpassswordLineEdit">
             <property name="inputMask">
              <string/>
             </property>
             <property name="text">
              <string>claire</string>
             </property>
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
            </widget>
           </item>
           <item row="4" column="0" colspan="2">
            <widget class="Line" name="line_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="arcstablenameLabel">
             <property name="text">
              <string>Input Table name</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QLineEdit" name="arcstablenameLineEdit">
             <property name="inputMask">
              <string/>
             </property>
             <property name="text">
              <string>brut_arcs</string>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="Line" name="line">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="directoryBox">
         <property name="title">
          <string>Directory to write files</string>
         </property>
         <widget class="QWidget" name="verticalLayoutWidget_5">
          <property name="geometry">
           <rect>
            <x>9</x>
            <y>19</y>
            <width>401</width>
            <height>161</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QPushButton" name="browsePushButton">
               <property name="text">
                <string>Browse...</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="directoryLineEdit">
               <property name="text">
                <string>/home/clairel/Codes/+sorties_morpheo</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="mainTab">
     <attribute name="title">
      <string>Process' parameters</string>
     </attribute>
     <widget class="QGroupBox" name="optionsBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>451</width>
        <height>111</height>
       </rect>
      </property>
      <property name="title">
       <string>Ways' constitution's options</string>
      </property>
      <widget class="QWidget" name="formLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>431</width>
         <height>82</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="optionsFormLayout">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
        </property>
        <property name="labelAlignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="methodLabel">
          <property name="text">
           <string>Method</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="methodComboBox"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="thresholdLabel">
          <property name="text">
           <string>Threshold</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QDoubleSpinBox" name="thresholdDoubleSpinBox">
          <property name="value">
           <double>60.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="bufferLabel">
          <property name="text">
           <string>Buffer</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="BufferDoubleSpinBox">
          <property name="value">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="dropTablegroupBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>261</width>
        <height>51</height>
       </rect>
      </property>
      <property name="title">
       <string>Drop tables</string>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget_3">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>20</y>
         <width>220</width>
         <height>22</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="dropTableverticalLayout">
        <item>
         <widget class="QCheckBox" name="dropTABLESCheckBox">
          <property name="text">
           <string>SXYZ, SIF, ANGLES, INFO, VOIES</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="createTablegroupBox">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>30</y>
        <width>131</width>
        <height>91</height>
       </rect>
      </property>
      <property name="title">
       <string>More tables</string>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>117</width>
         <height>61</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="createTableVerticalLayout">
        <item>
         <widget class="QCheckBox" name="arcRueCheckBox">
          <property name="text">
           <string>Arcs and Rues</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="attributesGroupBox">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>10</y>
        <width>211</width>
        <height>221</height>
       </rect>
      </property>
      <property name="title">
       <string>More attributes</string>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>17</y>
         <width>201</width>
         <height>181</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="attributesVerticalLayout">
        <item>
         <widget class="QCheckBox" name="connexionCheckBox">
          <property name="text">
           <string>Connection</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="useCheckBox">
          <property name="text">
           <string>Use</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="inclusionCheckBox">
          <property name="text">
           <string>Inclusion</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="gradientCheckBox">
          <property name="text">
           <string>Gradient</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="localAccesscheckBox">
          <property name="text">
           <string>Local Accessibility</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QPushButton" name="calculatePushButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>GO !</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="classificationTab">
     <attribute name="title">
      <string>Tools</string>
     </attribute>
     <widget class="QGroupBox" name="actionGroupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>241</width>
        <height>201</height>
       </rect>
      </property>
      <property name="title">
       <string>Action type</string>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget_4">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>19</y>
         <width>221</width>
         <height>152</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QRadioButton" name="additionRadioButton">
          <property name="text">
           <string>Addition</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="soustractionRadioButton">
          <property name="text">
           <string>Soustraction</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="multiplicationRadioButton">
          <property name="text">
           <string>Multiplication</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="divisionRadioButton">
          <property name="text">
           <string>Division</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="absoluteDiffRadioButton">
          <property name="text">
           <string>Absolute difference</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="classificationRadioButton">
          <property name="text">
           <string>Classification</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QPushButton" name="modifyPushButton">
      <property name="geometry">
       <rect>
        <x>574</x>
        <y>10</y>
        <width>271</width>
        <height>141</height>
       </rect>
      </property>
      <property name="text">
       <string>GO !!</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="commonAttributesBox">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>10</y>
        <width>281</width>
        <height>201</height>
       </rect>
      </property>
      <property name="title">
       <string>Input informations</string>
      </property>
      <widget class="QWidget" name="formLayoutWidget_3">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>19</y>
         <width>267</width>
         <height>171</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="databaseFormLayout_2">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
        </property>
        <property name="labelAlignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="inputatt1Label">
          <property name="text">
           <string>Input attribute 1</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="inputatt1LineEdit">
          <property name="inputMask">
           <string/>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="resultattLabel">
          <property name="text">
           <string>Result attribute's name</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="resultattLineEdit">
          <property name="inputMask">
           <string/>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="inputatt2Label">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Input attribute 2</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="inputatt2LineEdit">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="inputMask">
           <string/>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QSpinBox" name="classnbSpinBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="maximum">
           <number>1000000000</number>
          </property>
          <property name="singleStep">
           <number>100</number>
          </property>
          <property name="value">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="classnbLabel">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Classes number</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QCheckBox" name="descentCheckBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Descendent</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="orderLabel">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Order</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="loggerBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>851</width>
      <height>331</height>
     </rect>
    </property>
    <property name="title">
     <string>Logs output</string>
    </property>
    <widget class="QTextBrowser" name="loggerTextBrowser">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>831</width>
       <height>281</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="debugCheckBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>429</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Debug level</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>873</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
